### PR TITLE
New builds for 1.47.x

### PR DESCRIPTION
**Pull request by h-vetinari**
I didn't know until I tried https://github.com/conda-forge/bazel-feedstock/pull/156 that there was a separate feedstock involved in grpc + bazel.

Obviously it needs to move along with the current migrations (shouldn't we move the building of the plugin to the grpc feedstock...? 🤔)

In any case, this feedstock needs a boost (new branch for 1.46, new builds for 1.48, etc.), and since I'm trying to handle those migrations, I might as well do it.

### CODE REVIEW

**Comment by h-vetinari** [comment]
PTAL @xhochy, this is necessary to keep bazel moving; I'd be happy to join as maintainer here

**Comment by h-vetinari** [comment]
@hmaarrfk 
Since Uwe is the only maintainer here and pretty busy these days, would you mind merging this? Then I'll be a maintainer too and take care of the rest.

File: recipe/conda_build_config.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+# linux-aarch64 & linux-ppc64le built on linux-64
+# osx-arm64 built on osx-64
+bazel_arch:
+  - linux-x86_64   # [linux]
+  - darwin-x86_64  # [osx]
+
+bazel_sha:
+  - cbccf5c02d404b4d3fdcc347e8cf1fc9a40e232905a1a8745499a82bdfaf00af  # [linux]
+  - df204b5fdd93571eeb9aea48b8d2328ae39ed0f27f137c5c0d9281d6145d2f21  # [osx]
```
Comment (hmaarrfk) [inline recipe/conda_build_config.yaml:12]: is it really easier to add things here rather than to the meta.yaml file? Seems like added indirection.

Comment (h-vetinari) [reply recipe/conda_build_config.yaml:12]: Personal preference I guess... Thanks!

Comment (hmaarrfk) [reply recipe/conda_build_config.yaml:12]: Thanks for pushing things through!

**Pull request merged**